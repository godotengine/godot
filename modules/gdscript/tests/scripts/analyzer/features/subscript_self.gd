# https://github.com/godotengine/godot/issues/43221
extends Node

func test():
	name = "Node"
	print(self["name"])
	self["name"] = "Changed"
	print(name)
