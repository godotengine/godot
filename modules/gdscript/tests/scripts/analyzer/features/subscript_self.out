GDTEST_OK
Node
Changed
